jep/223/README.adoc

```diff
@@ -218,6 +218,18 @@ For the most current discussions on these points (and more) please refer to link
 
 No existing permission types are being removed at this point, which will allow for existing plugins that make use of them to continue to function.
 Removing code references to the "dangerous" permissions is outside the scope of this JEP.
+Since the granting of the `Jenkins.CONFIGURE` is optional, administrators who find the new permission does not provide value for them can simply continue to grant `Jenkins.ADMINISTER` to users who need overall administrator access (`Jenkins.CONFIGURE` is implied by `Jenkins.ADMINISTER`).
+While we do not anticipate that this change will introduce any regressions, we do acknowledge that some cases may be found after the initial release that may result in confusing or broken behavior for some plugins (ex. a configure link is shown for some plugin, but clicking it results in a permission failure when actually accessing its management link).  
+Because of this, we (the JEP-223 sponsors) are committed to the following support policy:
+
+* For issues raised against Jenkins Core, we will review and fix (or provide justification for not fixing) problems related to this change
+* For issues raised against plugins that are bundled with Jenkins, we will review and fix (or provide justification for not fixing) problems related to this change
```

Comment:
There is no such thing in any sense that would be useful to anyone. Perhaps this?

```suggestion
* For issues raised against plugins that are installed by default through the setup wizard, we will review and fix (or provide justification for not fixing) problems related to this change
```